packages/core/contracts/oracle/implementation/OptimisticOracle.sol

```diff
@@ -523,4 +588,9 @@ contract OptimisticOracle is Testable, Lockable {
     function _getIdentifierWhitelist() internal view returns (IdentifierWhitelistInterface) {
         return IdentifierWhitelistInterface(finder.getImplementationAddress(OracleInterfaces.IdentifierWhitelist));
     }
+
+    // Stamps the ancillary data blob with the optimistic oracle tag denoting what contract requested it.
+    function _stampAncillaryData(bytes memory ancillaryData, address requester) internal pure returns (bytes memory) {
+        return abi.encodePacked(ancillaryData, "OptimisticOracle", requester);
```

Comment:
this seems good for now. we should propose an UMIP at some point soon that defines the expected structure of ancillary Data to standardize it.